src/server/pps/server/api_server.go

```diff
@@ -2232,7 +2235,7 @@ func (a *apiServer) rcPods(rcName string) ([]v1.Pod, error) {
 			Kind:       "ListOptions",
 			APIVersion: "v1",
 		},
-		LabelSelector: metav1.FormatLabelSelector(metav1.SetAsLabelSelector(labels(rcName))),
+		LabelSelector: metav1.FormatLabelSelector(metav1.SetAsLabelSelector(map[string]string{"app": rcName})),
```

Comment:
This is the bug fix, yea? Otherwise you'd specify the worker component and get no logs.

Reply:
Exactly, you'd get a pod not found error.